packages/search-types/src/response/response.types.ts

```diff
@@ -0,0 +1,10 @@
+import { TaggingRequest } from '../request';
+
+/**
+ * Response to be implemented by all responses that contain a part to track the show event.
+ *
+ * @public
+ */
+export interface TrackableShowResponse {
+  showTagging: TaggingRequest;
+}
```

Comment:
What would you think about renaming this file to `trackeable-show-response.model.ts`?

Reply:
Sure, you're right, that makes sense.